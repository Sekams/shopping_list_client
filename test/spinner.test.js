import React from 'react';
import Spinner from '../src/components/spinner'

test("renders the spinner component", () => {
    const wrapper = shallow(
        <Spinner />
    );
    expect(wrapper).toMatchSnapshot();
});

test("calls the showSpinner function", () => {
    const wrapper = shallow(
        <Spinner />
    );
    wrapper.instance().showSpinner(true);
    wrapper.instance().showSpinner(false);
    expect(wrapper).toMatchSnapshot();
});
