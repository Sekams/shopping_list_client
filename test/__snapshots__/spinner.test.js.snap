// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calls the showSpinner function 1`] = `
<div
  className="hide"
>
  <ul
    aria-busy="true"
    aria-label="Loading domino shop"
    className="spinner"
    role="progressbar"
  >
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
  </ul>
</div>
`;

exports[`renders the spinner component 1`] = `
<div
  className="hide"
>
  <ul
    aria-busy="true"
    aria-label="Loading domino shop"
    className="spinner"
    role="progressbar"
  >
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
    <li
      role="presentation"
    />
  </ul>
</div>
`;
